linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/package_pam_pkcs11_installed/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
```

Comment:
I'm worried about the renaming breaking existent tailorings.

Reply:
Exactly. Not only does it break backwards compatibility, it is a non-relevant change. It also breaks CCE mapping, and there are more packages in RHEL7 than just pkcs11 getting installed. This PR introduces more problems than it solves.

Reply:
Actually, latest version of STIG for RHEL7 now only requires `pam_pkcs11` package to be installed, that's why this rule now uses the `package_installed` template: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_7_security_technical_implementation_guide/V-72417

So the only problem here is the backwards compatibility issue

Reply:
Ummm.... no not true... CCE breaking is a problem and backwards compatibility is a problem. The appropriate fix is to have the OVAL generated.

Reply:
OK, can do it the other way around.
I see that tailoring compatibility can be an issue even for rules without OVAL - one may want to tailor them out exactly for this reason.

Reply:
This rule had OVAL check before, but since now it checks only for a specific package it didn't make sense to keep the custom OVAL, so I changed some time ago to use the `package_installed` template instead, but I didn't foresee that the OVAL wouldn't get generated... that's because the generated definition ID of the check was different from the rule's ID. But the template seems to be using the rule's id now: https://github.com/ComplianceAsCode/content/blob/master/shared/templates/template_OVAL_package_installed#L2 so the OVAL check is supposed to be correctly assigned to this rule.

Reply:
@matejak 
I've identified the issue. Actually we are still using the OVAL definition `package_pam_pkcs11_installed` in  https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_auth/oval/shared.xml#L5

And this definition was removed in https://github.com/ComplianceAsCode/content/commit/c73da05b65742ca6820a9b7c0fc9ee22097c805a#diff-52ce1eba7a9574ad3c71d2419debb827

So we have two options, revert back this change to have this extra OVAL generated or change this line https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_auth/oval/shared.xml#L5

to:
`      <extend_definition comment="pam_pkcs11 package is installed" definition_ref="install_smartcard_packages" />`

Reply:
Indeed, I think that the second option is better.

Reply:
I guess everything from here has been addressed.